### ISSUE TITLE
Server CI broken for a long time

### ISSUE THREAD

**Issue by advayDev1** [WorldHealthOrganization/app#1894]
https://github.com/WorldHealthOrganization/app/runs/1616128334

```
/bin/tar xz --warning=no-unknown-keyword -C /home/runner/work/_temp/942c869e-c8a4-44d4-b3a0-267ce30d17a2 -f /home/runner/work/_temp/42cc9b16-9ea8-441c-ba38-fd11144685bb
19
Error: Unable to process command '::add-path::/opt/hostedtoolcache/gcloud/321.0.0/x64/bin' successfully.
20
Error: The `add-path` command is disabled. Please upgrade to using Environment Files or opt into unsecure command execution by setting the `ACTIONS_ALLOW_UNSECURE_COMMANDS` environment variable to `true`. For more information see: https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/
21
Error: Unable to process command '::set-env name=CLOUDSDK_METRICS_ENVIRONMENT::github-actions-setup-gcloud' successfully.
22
Error: The `set-env` command is disabled. Please upgrade to using Environment Files or opt into unsecure command execution by setting the `ACTIONS_ALLOW_UNSECURE_COMMANDS` environment variable to `true`. For more information see: https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/
23
gcloud SDK installed without authentication.
```

**Comment by theswerd** [comment]
Thank you @advayDev1

**Comment by brunobowden** [comment]
Properly fixed by #1899:
https://github.com/WorldHealthOrganization/app/runs/1620698585
